packages/vega-typings/types/spec/encode.d.ts

```diff
@@ -81,27 +81,75 @@ export interface ColorHCL {
   l: NumericValueRef;
 }
 export interface GradientStop {
+  /**
+   * The offset fraction for the color stop, indicating its position within the gradient.
+   */
   offset: number;
-  color: string;
+  /**
+   * The color value at this point in the gradient.
```

Comment:
point -> stop?  (given the class name is stop)

Reply:
The stop defines a color at a point. I think it's right.